nss-day-cohort-48/rare-client-wanderering-wizards
# Added post form file and hooked it up to server

Steps to Test

1. Clone to local
2. Serve it up
3. Verify that create post button is present on the my posts page as well as all posts.
4. Click button and verify you are routed to the form field. 
5. Fill out form field and verify the request was successfully sent to the database and you are directed to the my posts page where your post should be visible.
